reporting/src/request/steps/check-context.js

```diff
@@ -13,6 +13,17 @@ export function checkValidContext(state) {
   if (!state.page || !state.tabUrlParts || !state.urlParts) {
     return false;
   }
+
+  // ignore requests that are related to page load but does not affect the page itself
+  // https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/webRequest/ResourceType#other
+  // examples:
+  // * link rel=search
+  // * link rel=manifest
+  // * link rel=prefetch
+  if (state.type === 'other') {
```

Comment:
A problem that I see is that it is extremely aggressive in filtering out requests.

Example: https://www.theguardian.com/football/2024/sep/30/bryan-zaragoza-osasuna-barcelona-la-liga

It makes a lot of requests like:
```
<link rel="manifest" href="https://assets.guim.co.uk/static/frontend/manifest.json" />
```
or
```
<script defer src="https://assets.guim.co.uk/polyfill.io/v3/polyfill.min.js?rum=0&features=es6%2Ces7%2Ces2017%2Ces2018%2Ces2019%2Cdefault-3.6%2CHTMLPictureElement%2CIntersectionObserver%2CIntersectionObserverEntry%2CURLSearchParams%2Cfetch%2CNodeList.prototype.forEach%2Cnavigator.sendBeacon%2Cperformance.now%2CPromise.allSettled&flags=gated&callback=guardianPolyfilled&unknown=polyfill&cacheClear=1"></script>
```

These will be then all ignored.